### DISCUSSION TITLE
MATLAB version of nnUNet

### DISCUSSION THREAD

**Discussion by dantesonmda** [MIC-DKFZ/nnUNet#1133] [General]
Hi,
Does anybody have a plan to make MATLAB version of nnUNet?

**Comment by FabianIsensee** [comment]
Not as far as I know

**Comment by FabianIsensee** [comment]
But why would you want that? Matlab is expensive as hell and barely used in the AI/machine learning community. Everything nnU-Net is based on comes for free :see_no_evil:
